recipes/pcapplusplus/cmake/conanfile.py

```diff
@@ -0,0 +1,60 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import copy, get, rmdir
+from conan.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.57.0"
+
+class PcapplusplusConan(ConanFile):
+    name = "pcapplusplus"
+    package_type = "static-library"
+    license = "Unlicense"
+    description = "PcapPlusPlus is a multiplatform C++ library for capturing, parsing and crafting of network packets"
+    topics = ("pcap", "network", "security", "packet")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/seladb/PcapPlusPlus"
+    settings = "os", "arch", "build_type", "compiler"
+    options = {
+        "immediate_mode": [True, False],
```

Comment:
```suggestion
        "immediate_mode": [True, False],
        "fPIC": [True, False],
```
I see no rule in Cmake file forcing or forbidden fpic 🤔

Reply:
I don't think `fPIC` is relevant for building static libraries, am I wrong? 🤔

Reply:
We manage fPIC always for static libraries in CCI. For shared we remove because fPIC should enabled always.

Reply:
Got it, updated here: https://github.com/conan-io/conan-center-index/pull/20296/commits/667883dd2315215e75910859d6ed6024f8ed63e5